imod/data/sample_data.py

```diff
@@ -248,3 +248,30 @@ def hondsrug_planar_river() -> xr.Dataset:
     planar_river["stage"] = (top - planar_river["stage"]) / 2 + planar_river["stage"]
 
     return planar_river
+
+
+def colleagues_river_data(path: Union[str, Path]):
+    """
+    River data with some mistakes introduced to showcase cleanup
+    utilities.
+    """
+    gwf_model = hondsrug_simulation(path)["GWF"]
+    dis_ds = gwf_model["dis"].dataset
+    riv_ds_old = gwf_model["riv"].dataset
+    # Existing RIV package has only layer coord with [1, 3, 5, 6]. This causes
+    # problems with some methods, at least with cleanup. Therefore align data
+    # here for the cleanup example. Not sure if we want to support river
```

Comment:
"Not sure if we want to support river packages with limited layer coords"

I'm not sure if we should put the above in the comment.

Instead you could also write:
"River packages with limited layer coords are currently not supported."

Reply:
I agree, I changed it to "River packages with limited layer coords are currently not fully supported.", as you can apparently write a functional model with the river package in only limited layers, but cannot call cleanup on it. I wasn't sure if we want to support that to begin with.